src/verify-and-receive.ts

```diff
@@ -15,6 +15,7 @@ export async function verifyAndReceive(
   // verify will validate that the secret is not undefined
   const matchesSignature = await verify(
     state.secret,
+    // @ts-expect-error verify uses createHmac, which can take Strings and Buffers
```

Comment:
I'd rather not. It's better to fix this at the source

Reply:
see https://github.com/octokit/webhooks-methods.js/pull/211